packages/graph/hash_graph/lib/graph/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 //! The entity-graph query-layer for the HASH datastore
 
-#![feature(lint_reasons, once_cell)]
+#![feature(lint_reasons, once_cell, generic_associated_types)]
```

Comment:
For us, this is not really adding features, we could workaround GATs, but it simplifies our pool code, see below